mne/channels/channels.py

```diff
@@ -434,29 +434,15 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         :func:`mne.viz.plot_trans`.
 
         .. versionadded:: 0.12.0
-
         """
         from ..viz.utils import plot_sensors
         return plot_sensors(self.info, kind=kind, ch_type=ch_type, title=title,
                             show_names=show_names, ch_groups=ch_groups,
                             axes=axes, block=block, show=show)
 
+    @copy_function_doc_to_method_doc(anonymize_info)
     def anonymize(self):
-        """Anonymize measurement information in place by removing
-        'subject_info', 'meas_date', 'file_id', 'meas_id' if they exist in
-        ``info``.
-
-        Returns
-        -------
-        self : instance of Raw | Epochs | Evoked
-            The data container.
-
-        Notes
-        -----
-        Operates in place.
-
-        .. versionadded:: 0.13.0
-        """
+        """.. versionadded:: 0.13.0"""
```

Comment:
I think this probably still needs to go in `Notes` section, it doesn't render properly:

https://2095-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/generated/mne.io.Raw.html#mne.io.Raw.anonymize